workflow-runtime/src/test/java/com/squareup/workflow/internal/RealRenderContextTest.kt

```diff
@@ -140,6 +140,7 @@ class RealRenderContextTest {
     val handler = context.onEvent<String> { expectedUpdate }
     assertTrue(eventActionsChannel.isEmpty)
 
+    context.freeze()
```

Comment:
Seems unrelated. Opportunistic cleanup?

Reply:
Nope, required because above change makes `onEvent` delegate to `send`, which will throw if we don't freeze the context before sending to it. I also want to make `onEvent` an extension method on the context.